### DISCUSSION TITLE
Nested Layout (Sublayout) Support

### DISCUSSION THREAD

**Discussion by mazataza** [qtile/qtile#2411] [Ideas]
I am new Qtile but i have used Xmonad for a while, what i missed on qtile is sublayout concept which is available in Xmonad.

is it possible with Qtile?

to understand what i mean:
in MonadTall layout there is a master-pane and a secondary-pane. the first window will be place to master and other windows will place vertically in the pane. in Xmonad we can define a sublayout which then can be used to layout those windows (for example all windows in the  secondary-pane will be displayed with tab layout).

**Comment by tych0** [comment]
I vaguely remember seeing patches about this somewhere, but nothing
ever went upstream.

**Comment by ramnes** [comment]
I think that what you're looking for is currently discussed in another thread (see the part about the Slice layout): #2412

**Reply by mazataza** [reply]
The slice layout may be usefull in the example i described above. but it can place a specific match on the slice and in the other part it use the fallback layout.
in Xmonad the sublayout is more feature rich. it maybe usefull if you guys take a look to it.
in Xmonad you can move windows in and out to sublayout.
